src/RotateTransform.ts

```diff
@@ -18,5 +18,7 @@ export class RotateTransform extends Transform {
         }
         return retVal;
     }
+    
+    get Rotation() : object { return this.getRotation(); }
```

Comment:
Looks good, but curious of the utility - is this too keep with an accessor standard in the rest of the codebase?

Reply:
Yes, the existing code  for Presentation 3 manifests uses getPropertyName() for accessors, so the 3D extensions provides this as well. As a personal preference I prefer the .PropertyName accessor, so I added this for Rotation. Admittedly this is inconsistent and does puff up the source code a bit.